Segment fault when minio attempt to send message to Kafka

Ran into a SIGSEGV: segmentation violation when Minio attempt to send a message to Kafka

The stack track trace is

```
   .NET:       https://docs.min.io/docs/dotnet-client-quickstart-guide
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x14ab5aa]

goroutine 2223 [running]:
github.com/minio/minio/pkg/event/target.(*KafkaTarget).send(0xc0004b2c60, 0x1f43c1d, 0x3, 0x1f4bdd8, 0x8, 0x0, 0x0, 0xc001b28240, 0x18, 0xa, ...)
	github.com/minio/minio/pkg/event/target/kafka.go:180 +0x2ea
github.com/minio/minio/pkg/event/target.(*KafkaTarget).Save(0xc0004b2c60, 0x1f43c1d, 0x3, 0x1f4bdd8, 0x8, 0x0, 0x0, 0xc001b28240, 0x18, 0xa, ...)
	github.com/minio/minio/pkg/event/target/kafka.go:158 +0x118
github.com/minio/minio/pkg/event.(*TargetList).Send.func1.1(0xc00132de10, 0xc0001d7080, 0xc00166ef00, 0xc00133a82f, 0x6, 0xc00133a836, 0x5, 0x22b3ac0, 0xc0004b2c60)
	github.com/minio/minio/pkg/event/targetlist.go:137 +0x155
created by github.com/minio/minio/pkg/event.(*TargetList).Send.func1
	github.com/minio/minio/pkg/event/targetlist.go:134 +0x231

```


## Your Environment
Minio running on docker with latest image minio/minio

From docker inspect 
```        "Id": "cdcde88d35c168b2a4b6df899fdf4172c2b4577886bab06ea8c6f66466492da3",
        "Created": "2021-02-05T17:01:06.314305419Z",
      "Image": "sha256:9d30bccd56c3adc4eae260392db59a6a17a9b557c5ae1be49ab7ca197ff8d9ed",
```

on AWS EC2 T2 X Large

## Time this occured

12:01:13 UTC 02/18/2021

--- a/pkg/event/target/kafka.go
+++ b/pkg/event/target/kafka.go
@@ -160,6 +160,9 @@ func (target *KafkaTarget) Save(eventData event.Event) error {
 
 // send - sends an event to the kafka.
 func (target *KafkaTarget) send(eventData event.Event) error {
+	if target.producer == nil {
+		return errNotConnected
+	}
 	objectName, err := url.QueryUnescape(eventData.S3.Object.Key)
 	if err != nil {
 		return err

